### ISSUE TITLE
Add description

### ISSUE THREAD

**Issue by ThinkDigitalSoftware** [isoos/elastic_client#12]
Can you add a description to the package so we know what it's for? I was about to share it, but there's no info to help the dev know what it is and how to use it

**Comment by isoos** [comment]
@ThinkDigitalSoftware You mean more content in the README? Yeah, that is a bit lacking right now...

**Comment by ThinkDigitalSoftware** [comment]
Yes please :)

**Comment by vivekaion** [comment]
yes please, I am new and can't build complex query. how to build a complex query?

**Comment by vivekaion** [comment]
if you could show me some way to build query like following, I am willing to contribute to your repository, I am new to Dart, but working with elastic from many years.

my question is this.



`My user search
"the avenger infinity war"

so i need complex query in three part
Part 1: importance 1.5
		phrase search : "the avenger infinity war"

PArt 2: importance 1.0
		Word search "the" and "avenger" and "infinity" and "war"
		
PArt 3: importance 0.5
        Fuzzy search		
		"the"~0.5 and 
		"avenger"~0.5 and 
		"infinity"~0.5 and 
		"war"~0.5
		
	combining this search to one. well i am willing to contibute if you show me some way to build this, in string version i can made this, but don't know how to build in Map<dynamic,dynamic>
	
	is there any way, just a small hint will also be much helpful.
		`
